setup/tue.bash_functions

```diff
@@ -418,6 +418,13 @@ function tue-get
         return 1
     fi
 
+    #Create btrfs snapshot if possible:
```

Comment:
I am not really familiar with btrfs, but now this is done with every tue-get function. I think that is unwanted. If I am wrong, please explain.

Reply:
I think it is exactly wanted to make a backup at every tue-get action. Otherwise which moments would you select would be wise? 

In btrfs backups are cheap by the way. They do not cost any time. And if nothing has changed, they do not cost any space. So running tue-get update 20 times in a row has no disadvantage ;-)

Reply:
But now it is also done when you tue-get dep etc., right? I think that is unwanted. So only with install, update and remove. Do you agree?

Reply:
OK, that makes sense :+1: